changelog/207.breaking.rst

```diff
@@ -0,0 +1 @@
+Adding the updated and better version of the Coalignment API.
```

Comment:
This changelog when this PR is done will need actual details and links to documentation or examples.